### PR TITLE
feat(dropdown): allow sub text in the dropdown list

### PR DESCRIPTION
**Pull request by realdreamer**
Reviewers: @MLKiiwy @kagnamazian @realdreamer @galagmon @Maruceau @gsbkr @michelvdhoek

Preview: https://static-staging.usabilla.com/stylabilla/pr/blup-64-dropdown/docs/section-components.html#kssref-components-dropdown-subtext

Note: @galagmon @Maruceau - we are not modifying the existing dropdown to a new design. we are only adding the flexibility to support sub-text in the dropdown items for now.

### CODE REVIEW

File: docs-builder/kss-assets/kss.scss [hunk 1]
```diff
@@ -868,6 +868,10 @@ $kss-vertical-rhythm      : $kss-font-size * 1.5;
   height: 185px;
 }
 
+#kssref-components-dropdown-subtext .kss-modifier__wrapper {
+  height: 400px;
```
[the remark this answers is not in the document]
Comment (realdreamer) [reply docs-builder/kss-assets/kss.scss]: this style is for style guide. so we can't use our stylabilla variables here.

**Comment by galagmon** [comment]
That is fine by me 👍 
Question: is the keyboard navigation only in react stylabilla

**Comment by realdreamer** [comment]
@galagmon - yeah. keyboard navigation works only in react-stylabilla.

**Comment by galagmon** [comment]
@realdreamer we are missing line-height there

**Pull request merged**